lib/formulator/input.ex

```diff
@@ -43,10 +43,12 @@ defmodule Formulator.Input do
   defp add_format_validation_attribute(options, _, _), do: options
 
   defp option_enabled?(options, field, default) do
-    Enum.any?([
-      options[field] == true,
-      Application.get_env(:formulator, field, default) == true,
-    ])
+    cond do
+      Keyword.get(options, field) == false -> false
+      Keyword.get(options, field) == true -> true
+      Application.get_env(:formulator, field, default) == true -> true
+      true -> false
```

Comment:
Minor: what do you think about using `_` here instead of `true`. With all the lovely boolean logic here, that last part was a bit hard to follow for me.

```suggestion
      _ -> false
```
Do or do not, the choice is yours.

Reply:
Ya, sounds good.